prow/config/config.yaml

```diff
@@ -358,7 +358,7 @@ presubmits:
   - name: gofmt
     branches:
     - main
-    run_if_changed: '\.go$'
+    run_if_changed: '((\.go)|^hack/gofmt.sh)$'
```

Comment:
Nit: the `.` in `gofmt.sh` will match correctly. But probably you wanted to escape the `.` with `\.`.

Reply:
thanks for the catch